.ddev/config.yaml

```diff
@@ -0,0 +1,18 @@
+type: drupal10
+docroot: web
+php_version: "8.1"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames: []
+additional_fqdns: []
```

Comment:
```suggestion
```

Removing these will make projects (and this repo) easier to maintain.

Reply:
These are things that people rarely change. And of course, just running `ddev config` will fully populate all this again. What I'm trying to say is, "If you're going to add a .ddev/config.yaml, only add value in it". In general, anybody who uses DDEV knows they can do a `ddev config --auto` and they'll already get all this done for them.

Reply:
That said, this is just a matter of personal preference, my usual recommendation. It's not critical.

Reply:
FWIW, I agree with removing these. Were they ever to change in DDEV, this file would need to be updated to new values. If they aren't included, that would just happen in the background. I think it makes this project easier to maintain in the long run. Also, if it turns out this is a big problem with new users, then this can be changed later to add these variables back in.